print-rt6.c

```diff
@@ -33,13 +33,118 @@
 
 #include "ip6.h"
 
+int
+rt6_tlv_print(netdissect_options *ndo, const u_char *p, u_int bytes_left)
```

Comment:
The function is related to SRH and will be called for SRH only (case IPV6_RTHDR_TYPE_4).

so i think it should be renamed to srh_tlv_print ()